lib/python/vdm/server/HTTPListener.py

```diff
@@ -2241,13 +2241,20 @@ def get(database_id):
                     has_stopped = True
                 serverDetails.append({server[0]['hostname']: response.json()})
 
-            if has_stalled:
-                status.append({'status': 'stalled'})
-            elif has_run == True and has_stopped:
-                status.append({'status': 'stalled'})
-            elif not has_stalled and not has_stopped and has_run:
+            # if has_stalled:
+            #     status.append({'status': 'stalled'})
+            # elif has_run == True and has_stopped:
+            #     status.append({'status': 'stalled'})
+            # elif not has_stalled and not has_stopped and has_run:
+            #     status.append({'status': 'running'})
+            # elif has_stopped and not has_stalled and not has_run:
+            #     status.append({'status': 'stopped'})
+
+            if has_run == True:
                 status.append({'status': 'running'})
-            elif has_stopped and not has_stalled and not has_run:
+            elif has_stalled == True and not has_run:
```

Comment:
It is always 'not has_run' when we get here, so you don't need to have that condition here.